### ISSUE TITLE
'Preventing download of vcredist to solve installation issues' makes little sense and seems false

### ISSUE THREAD

**Issue by LinuxOnTheDesktop** [rjpcomputing/luaforwindows#133]
Thank you for providing lua for Windows.

Your download page has this sentence (or rather 'sentence'): 'Preventing download of vcredist to solve installation issues.' What is that trying to say? That the build at issue solves installation problems by preventing the download of 'vcredist'? If that is what is meant (and surely you should improve the sentence) then  the sentence seems false. For, I installed the build, and the installer installed a Visual C library; and, nonetheless, my Lua programs fail to run, giving instead this error:

`:1: unexpected symbol near ' '`

Previously, I had luaForWindows installed, along with a couple of Visual C libraries, and lua worked. I removed the libraries, found that lua no longer worked, and that is when I reinstalled lua - with the above results. Moreover, just now I installed three separate versions of the Visual C libraries and lua is still not working.

**Comment by LinuxOnTheDesktop** [comment]
The 'unexpected symbol' problem owed to an encoding problem (that may in turn have owed to changes with the editor I used). I solved the problem by ensuring my Lua scripts were saved with what Sublime Text calls 'UTF-8' encoding. (Or rather I suspect all that. I don't know for sure, because I abandoned LuaForWindows for a different way of running Lua.)

It remains the case that the sentence on your download page is rather hard to understand - but I will close this issue.
